p2panda-spaces/src/message.rs

```diff
@@ -139,20 +170,29 @@ where
     fn from_both_args(
         group_id: ActorId,
         auth_action: AuthGroupAction<C>,
+        auth_dependencies: Vec<OperationId>,
         encryption_control_message: EncryptionControlMessage,
+        encryption_dependencies: Vec<OperationId>,
         direct_messages: Vec<EncryptionDirectMessage>,
     ) -> Self {
         let control_message = match (auth_action, encryption_control_message) {
             (
                 AuthGroupAction::Create { initial_members },
                 EncryptionControlMessage::Create { .. },
             ) => ControlMessage::Create { initial_members },
+            (AuthGroupAction::Add { member, access }, EncryptionControlMessage::Add { .. })
+                if !access.is_pull() =>
+            {
+                ControlMessage::Add { member, access }
+            }
```

Comment:
We would panic here if we're adding a member with only "pull" access, right? Wouldn't the correct behaviour be to _always_ return the `Add` control message? On the level of space messages we don't distinct between encryption and auth.